src/main/kotlin/no/nav/familie/tilbake/behandlingskontroll/BehandlingskontrollService.kt

```diff
@@ -128,6 +134,19 @@ class BehandlingskontrollService(
         }
     }
 
+    @Transactional
+    fun behandleBrevmottakerSteg(behandlingId: UUID) {
+        behandlingsstegstilstandRepository.findByBehandlingIdAndBehandlingssteg(
+            behandlingId,
+            Behandlingssteg.BREVMOTTAKER
+        ) ?.apply {
+            oppdaterBehandlingsstegsstaus(behandlingId, Behandlingsstegsinfo(Behandlingssteg.BREVMOTTAKER, AUTOUTFØRT))
```

Comment:
Lurer litt på om man skal flytte disse to loglinjene inn til oppdaterBehandlingsstegsstaus og opprettBehandlingsstegOgStatus istedenfor. Føler at det er mer riktig at det er de metodene som logger dette, slik at ansvaret på logging ikke ligger i metoden før. Blir litt mer cleanere kode her også mtp at man slipper .also til slutt.

Reply:
Gjorde som i BehandleVergeSteg-funksjonen over og logger i forkant istedenfor :)